src/style.rs

```diff
@@ -21,6 +21,12 @@ pub struct Style {
     /// The style's background color, if it has one.
     pub background: Option<Color>,
 
+    // The style's os control type, if it has one.
+    // Used by corresponding public API functions in
+    // AnsiGenericString. This allows us to keep the
+    // prefix and suffix bits in the Style definition.
+    pub(crate) oscontrol: Option<OSControl>,
+
```

Comment:
It slipped through the cracks but adding this as a `pub(crate)` breaks the public API that previously allowed to construct `Style` directly (or to destructure it in totality). 

See #46 for what we should do about this.

cc @mhelsley